docs/deployment/postgres-database-deployment.md

```diff
@@ -0,0 +1,353 @@
+
+- [Introduction to Terraform](https://www.terraform.io/intro/index.html)
+- [Getting Started with Terraform and AWS](https://learn.hashicorp.com/terraform/?track=getting-started#getting-started)
+- [Terraform Configuration Language](https://www.terraform.io/docs/configuration/index.html)
+
+For Cumulus specific instructions on installation of Terraform, refer to the main [Cumulus Installation Documentation](https://nasa.github.io/cumulus/docs/deployment/deployment-readme#install-terraform)
+
+#### Aurora/RDS
+
+This document also assumes some basic familiarity with Postgres databases, and Amazon Aurora/RDS.   If you're unfamiliar consider perusing the [AWS docs](https://aws.amazon.com/rds/aurora/), and the [Aurora Serverless V1 docs](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/aurora-serverless.html).
+
+## Prepare deployment repository
+
+ > If you already are working with an existing repository that has a configured `rds-cluster-tf` deployment for the version of Cumulus you intend to deploy or update, *or*  just need to configure this module for your repository, skip to [Prepare AWS configuration.](postgres_database_deployment#prepare-aws-configuration)
+
+Clone the [`cumulus-template-deploy`](https://github.com/nasa/cumulus-template-deploy) repo and name appropriately for your organization:
+
+```bash
+  git clone https://github.com/nasa/cumulus-template-deploy <repository-name>
+```
+
+We will return to [configuring this repo and using it for deployment below](#deploying-the-cumulus-instance).
+
+<details>
+  <summary>Optional: Create a new repository</summary>
+
+  [Create a new repository](https://help.github.com/articles/creating-a-new-repository/) on Github so that you can add your workflows and other modules to source control:
+
+```bash
+  git remote set-url origin https://github.com/<org>/<repository-name>
+  git push origin master
+```
+
+You can then [add/commit](https://help.github.com/articles/adding-a-file-to-a-repository-using-the-command-line/) changes as needed.
+
+> **Note**: If you are pushing your deployment code to a git repo, make sure to add `terraform.tf` and `terraform.tfvars` to `.gitignore`, **as these files will contain sensitive data related to your AWS account**.
+
+</details>
+
+---
+
+## Prepare AWS configuration
+
+To deploy this module, you need to make sure that you have the following steps from the [Cumulus deployment instructions](deployment-readme#prepare-aws-configuration) in similar fashion *for this module*:
+
+- [Set Access Keys](deployment-readme#set-access-keys)
+- [Create the state bucket](deployment-readme#create-the-state-bucket)
+- [Create the locks table](deployment-readme#create-the-locks-table)
+
+--
+
+### Configure and deploy the module
+
+These steps should be executed in the `rds-cluster-tf` directory of the template deploy repo that you previously cloned. Run the following to copy the example files.
```

Comment:
Add a note above here to reinforce that the cluster only needs to be deployed once per account, not continuously like the data-persistence and cumulus modules

Reply:
37c4c2062